Gemfile

```diff
@@ -93,7 +93,7 @@ gem 'retry_block', '~> 1.2.0'
 gem 'colorize', '~> 0.8.1'
 gem 'dogstatsd-ruby', '~> 3.2.0'
 gem 'http', '~> 4.0'
-gem 'robots_tag_parser', '~> 0.1.0', git: 'https://github.com/GSA/robots_tag_parser'
```

Comment:
The unnecessary git ref was leftover from before we released `robots_tag_parser` as a gem.